sdb/commands/linux/stacks.py

```diff
@@ -455,3 +455,129 @@ def pretty_print(self, objs: Iterable[drgn.Object]) -> None:
     def no_input(self) -> Iterable[drgn.Object]:
         self.validate_context()
         yield from [sdb.get_prog().crashed_thread().object]
+
+class ArgStack(sdb.Locator, sdb.PrettyPrinter):
+    """
+    Print a stack with arguments.
+
+    EXAMPLES
+    sdb> echo 0xffff96609b7d4680 | stack_args
+    (struct task_struct *)0xffff96609b7d4680
+    #0  struct rq *context_switch(struct rq *rq, struct task_struct *prev, struct task_struct *next, struct rq_flags *rf)
+          rf = (struct rq_flags *)<absent>
+          next = (struct task_struct *)<absent>
+          prev = (struct task_struct *)<absent>
+          rq = (struct rq *)<absent>
+    #1  void __schedule(bool preempt)
+          preempt = (bool)<absent>
+          prev = (struct task_struct *)0xffff96609b7d4680
+          next = (struct task_struct *)<absent>
+          switch_count = (unsigned long *)<absent>
+          rf = (struct rq_flags){ 0, {}, 4 }
+          rq = (struct rq *)<absent>
+          cpu = (int)<absent>
+    #2  void schedule()
+          tsk = (struct task_struct *)<absent>
+    ...
+    """
+
+    names = ["stack_args"]
```

Comment:
other names we might consider for this:
* `locals` (it prints local variables)
* `stack --args` (extension to existing command rather than new command)